src/vl-search-filter.js

```diff
@@ -57,8 +61,29 @@ export class VlSearchFilter extends NativeVlElement(HTMLDivElement) {
     return 'vl-search-filter';
   }
 
+  get _form() {
+    return this.querySelector('form');
+  }
+
+  get _slotForm() {
+    const slot = this.querySelector('slot');
+    if (slot) {
+      return slot.assignedElements().find((element) => element.tagName == 'FORM');
+    }
+  }
+
+  get _hasForm() {
+    return !!this.querySelector('form');
+  }
+
   get formData() {
-    return new FormData(this.querySelector('form'));
+    if (this._hasForm) {
+      return new FormData(this._form);
+    } else {
+      const form = document.createElement('form');
+      form.append(... this._slotForm.children);
```

Comment:
is da performant om da elke keer te doen?

Reply:
Het is bij het asynchroon verwerken van het opvragen van de formData, dus ik verwacht niet meteen performantie problemen. Ik heb het niet getest op performantie.

Reply:
Ik heb de implementatie ondertussen gewijzigd waardoor er geen nieuwe form aangemaakt wordt.